sdk/inc/azure/iot/az_iot_adu_client.h

```diff
@@ -373,7 +382,7 @@ typedef struct
 typedef struct
 {
   /**
-   * Identity of a file, referenced in the update request.
+   * Identity of a file, correlated with the same id in the update request.
```

Comment:
Nit: correlated with the `id` in the update request. 
(Unclear why it's the "same".)

Reply:
Or we could say it _can_ be correlated with the `id` in the update request?

Reply:
Also: can we strongly reference the type for the update request?